src/test/java/com/google/javascript/clutz/interface.d.ts

```diff
@@ -1,4 +1,8 @@
 declare namespace ಠ_ಠ.clutz {
+  namespace interface_exp {
+    function staticMethod ( ) : number ;
```

Comment:
isn't this basically the same test as 'interface_static_method.d.ts'. Let's just keep one of the two.

Reply:
done.